### PR TITLE
 fix: remove cy.viewport limits

### LINKED ISSUE #5892: Remove upper limit on cy.viewport; investigate removing lower limit
### Current behavior:

`cy.viewport` has limited bounds (minimum 20px, maximum 8000px)

### Desired behavior:

* `cy.viewport` has an upper bound of 100k (unlikely to be practically useful, but will prevent users from accidentally entering a billion or something
* Investigate if it's necessary to protect users with an upper limit - does `cy.viewport(1 million, 1 million)` actually cause performance issues? If not we can remove the upper limit altogether.
* Investigate if we need the lower limit - it may be that some applications are suited to a 0x0 viewport, like a pixel tracker that works in an `<iframe>`.

### PR DESCRIPTION
**Pull request by jabczyk**
- Closes #5892

### User facing changelog
Removed 20px-8000px bounds for `cy.viewport()`. It now supports range from 0 to any finite number.

### Additional details
- Tracking pixel tests don't need a 20x20px viewport. 0x0px is enough.
- Large viewports don't seem to cause any performance issues, so there's no need to prevent them.

### How has the user experience changed?
Users are now able to create 0x0px viewports.
```js
// before 
cy.viewport(20, 20)
cy.visit('tracking-pixel-page')

// after
cy.viewport(0, 0)
cy.visit('tracking-pixel-page')
```
Users are now able to test applications with huge UI.
```js
cy.viewport(5 * 1920, 1080)
```

### PR Tasks

- [X] Have tests been added/updated?
- [x] Has the original issue been tagged with a release in ZenHub? 
- [X] Has documentation PR? cypress-io/cypress-documentation#2288

### CODE REVIEW

**Comment by flotwig** [comment]
Was curious what this looked like.

0x0 viewport: 
image

100k x 100k viewport:
image

File: packages/driver/src/cy/commands/window.coffee [hunk 1]
```diff
@@ -159,7 +159,7 @@ module.exports = (Commands, Cypress, cy, state, config) ->
 
       widthAndHeightAreWithinBounds = (width, height) ->
         _.every [width, height], (val) ->
-          val >= 20 and val <= 8000
+          val >= 0
```
Comment (flotwig) [inline packages/driver/src/cy/commands/window.coffee]: This won't catch `Infinity` or `NaN`, maybe this:

```suggestion
          _.isFinite(val) && val >= 0
```

Comment (jabczyk) [reply packages/driver/src/cy/commands/window.coffee]: This is already checked with `widthAndHeightAreValidNumbers` (line 158, 201).

Comment (flotwig) [reply packages/driver/src/cy/commands/window.coffee]: Ah, I missed that, thanks

**Pull request merged**